src/components/main/itemSelectionModal/RowMenu.tsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from 'react';
+
+import KeyboardArrowRightIcon from '@mui/icons-material/KeyboardArrowRight';
+import { Button, IconButton, Stack, Typography, styled } from '@mui/material';
+
+import { ItemType } from '@graasp/sdk';
+import { ItemIcon } from '@graasp/ui';
+
+import {
+  buildItemRowArrowId,
+  buildNavigationModalItemId,
+} from '@/config/selectors';
+
+import { NavigationElement } from './Breadcrumbs';
+
+export interface RowMenuProps {
+  item: NavigationElement;
+  onNavigate: (item: NavigationElement) => void;
+  selectedId?: string;
+  onClick: (item: NavigationElement) => void;
+  isDisabled?: (item: NavigationElement) => boolean;
+}
+
+const StyledButton = styled(Button)<{ isSelected: boolean }>(
+  ({ theme, isSelected }) => ({
+    color: theme.palette.text.primary,
+    width: '100%',
+    justifyContent: 'start',
+    background: isSelected ? theme.palette.grey[200] : 'none',
+    textTransform: 'none',
+    pl: 1,
+  }),
+);
+
+const RowMenu = ({
+  item,
+  onNavigate,
+  onClick,
+  selectedId,
+  isDisabled,
+}: RowMenuProps): JSX.Element | null => {
+  const [isHoverActive, setIsHoverActive] = useState(false);
```

Comment:
You need a state for hover ??

Reply:
Actually yes I think, because of the disabled it was pushed above, and stack does not handle hover. Plus we need the variable to show or not the button. I guess we could achieve it with some complex css but well..